### PR TITLE
Update Cypress@8.2.0

### PR DESCRIPTION
**Pull request by anselmbradford**
## Changes

- Update to Cypress@8.2.0
- Refresh npm cache

## How to test this PR

1. PR checks should pass.

### CODE REVIEW

File: docker/cypress/Dockerfile [hunk 1]
```diff
@@ -34,7 +34,7 @@ RUN yum update -y && \
         nss* \
         alsa-lib* && \
     yum clean all && rm -rf /var/cache/yum && \
-    npm install -D cypress@8.1.0 && \
+    npm install -D cypress@8.2.0 && \
```
Comment (contolini) [inline docker/cypress/Dockerfile:37]: If we [globally install yarn](https://github.com/cfpb/consumerfinance.gov/blob/c4a515d472af59f755d7b494af5363f26709b2a3/docker/cypress/Dockerfile#L22) should we be using it to install and run cypress?

Comment (anselmbradford) [reply docker/cypress/Dockerfile]: Works for me https://github.com/cfpb/consumerfinance.gov/pull/6571/commits/8e64f1d15896ca51e82ea38a0cf489e1159aae10

Comment (anselmbradford) [reply docker/cypress/Dockerfile]: Does `$(npm bin)/cypress verify` need to be updated? I not so sure what that is doing

Comment (willbarton) [reply docker/cypress/Dockerfile]: In fact, we should remove the Cypress Dockerfile entirely at this point.

**Review by wpears — APPROVED** [review]

**Pull request merged**